package.json

```diff
@@ -1,35 +1,30 @@
 {
   "name": "level-js",
   "version": "3.0.0-rc1",
-  "description": "leveldown/leveldb library for browsers using IndexedDB",
+  "description": "An abstract-leveldown compliant store on top of IndexedDB",
+  "author": "max ogden",
+  "license": "MIT",
   "main": "index.js",
   "scripts": {
     "test": "standard && airtap --local --no-coverage test/index.js",
-    "test-browsers": "standard && airtap --sauce-connect --loopback airtap.local --no-coverage test/index.js"
+    "test-browsers": "standard && airtap --sauce-connect --loopback airtap.local --no-coverage test/index.js",
+    "remark": "remark README.md CONTRIBUTORS.md -o"
```

Comment:
How does `remark` know how each file should be processed and which plugin to apply to which output? `CONTRIBUTORS.md` is completely generated, right? Just curious about the algorithm of this command, in which order things are done etc.

Reply:
It's pretty much the same as doing `remark README.md && remark CONTRIBUTORS.md`. It'll use the same configuration, same list of plugins, but it creates a new processor (and plugin "instances") for each file.

Reply:
If you're wondering why `remark-git-contributors` doesn't modify `README.md`: because `README.md` doesn't have a `# Contributors` heading.

Reply:
Aah ok. So I guess this mean `CONTRIBUTORS.md` has to exist and have an empty `# Contributors` section at the very least.